layouts/shortcodes/blocks/feature.html

```diff
@@ -2,10 +2,12 @@
 {{ $url_text := .Get "url_text" -}}
 <div class="col-lg-4 mb-5 mb-lg-0 text-center">
 <div class="mb-4 h1">
+{{ with .Get "url" }}<a class="text-white" target="_blank" href="{{ . }}">{{ end }}
   <i class="{{ if not (or (hasPrefix $icon "fas ") (hasPrefix $icon "fab ")) }}fas {{ end }}{{ $icon }}"></i>
+</a>
 </div>
 <h4 class="h3">
-  {{- .Get "title" | markdownify -}}
+  <p>{{ with .Get "url" }}<a target="_blank" href="{{ .  }}">{{ end }}{{- .Get "title" | markdownify -}}</a></p>
```

Comment:
If no parameter `url` is given to the shortcode, we end up with this invalid markup:

```
<p>My title</a><p>
```

Can you please correct this?

Reply:
Done